opencog/reasoning/RuleEngine/rule-engine-src/pln/PLNCommons.cc

```diff
@@ -51,9 +51,8 @@ Handle PLNCommons::create_quoted(Handle himplicant)
     return hquoted;
 }
 
-Handle PLNCommons::create_bindLink(
-        Handle himplicant, bool vnode_is_typedv)
-                throw (opencog::InvalidParamException)
+Handle PLNCommons::create_bindLink(Handle himplicant, bool vnode_is_typedv)
+    throw (opencog::InvalidParamException)
```

Comment:
Yes, specifying the throw at the function declaration is probably useless at best, see

http://www.drdobbs.com/sutters-mill-a-pragmatic-look-at-except/184401544